packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -199,6 +199,57 @@ describe('Account', () => {
       expect(accountHistories.length).toStrictEqual(1)
     })
   })
+
+  it('should listAccountHistory mine with options txn', async () => {
```

Comment:
No wonder all the tests looks so weird to me, the `txn` is working as filter (https://github.com/DeFiCh/ain/pull/1110#discussion_r808961710)

As the PR  title suggested, the main purpose is a composite indexed offset `blockheight-txn`, perhaps should mandate `txn` must be used together with `maxBlockHeight`?

Reply:
@surangap the decision is to remove `filtration` and leave with only pagination token `height-txn`?

Reply:
@ivan-zynesis current plan - filtration will be removed and if the block height is not provided latest block will considered.